history/characters/17000_28000_bronzebeard.txt

```diff
@@ -24,7 +24,7 @@
 	trait=education_martial_4 trait=patient trait=generous trait=compassionate trait=honest
 	father=17042	#Horhunn
 	disallow_random_traits = yes
-	426.4.16={ birth=yes trait=creature_dwarf immortal_age=18 }
+	365.4.16={ birth=yes trait=creature_dwarf immortal_age=18 }
 	444.4.16={
```

Comment:
```suggestion
	383.4.16={
```

Reply:
Ok. I wasn't sure if those needed an update or not so I didn't change them.